benchmarks/src/sha3-rocc.c

```diff
@@ -41,13 +51,15 @@ int main() {
     for(i = 0; i < SHA3_256_DIGEST_SIZE; i++){
       printf("output[%d]:%d ==? results[%d]:%d \n",i,output[i],i,result[i]);
       if(output[i] != result[i]) {
-        printf("Outputs don't match!\n");
+        printf("Failed: Outputs don't match!\n");
+        printf("SHA execution took %lu cycles\n", end - start);
         return 1;
       }
     }
   } while(0);
 
-  
-  printf("success!\n");
+  printf("SHA execution took %lu cycles\n", end - start);
+
+  printf("Success!\n");
```

Comment:
if you just put this before the cycle count reporting, the test reference will be simple